src/app/common/modals/calendar-modal/calendar-modal.component.html

```diff
@@ -0,0 +1,69 @@
+<div class="calendar-modal-component mat-typography">
+
+  <!-- Dialog header -->
+  <div class="d-flex d-flex-row">
+    <div class="d-flex-grow">
+      <h2 mat-dialog-title>Web calendar</h2>
+    </div>
+    <mat-slide-toggle
+      #webcalToggle
+      [matTooltip]="webcal ? 'Disable the web calendar' : 'Enable the web calendar'"
+      matTooltipPosition="above"
+      [checked]="!!(webcal)"
+      [disabled]="working"
+      (click)="onWebcalToggle()">
+    </mat-slide-toggle>
+  </div>
+
+  <!-- Dialog content -->
+  <div mat-dialog-content>
+
+    <small>Your web calendar is currently {{ webcal ? 'enabled' : 'disabled' }}.</small>
+
+    <div *ngIf="!webcal">
+      <p>
+        The web calendar displays due dates of your tasks in an iCalendar that can be added to any iCalendar client
+        (e.g. Outlook/Google/iCloud calendar).
+      </p>
+    </div>
+
+    <div *ngIf="webcal">
+      <p>
+        Use the following URL to subscribe to your web calendar from your iCalendar client (e.g. Outlook/Google/iCloud
+        Calendar).
+      </p>
+      <div class="d-flex d-flex-row">
```

Comment:
Where is this coming from? We should be using fx-layout: https://github.com/angular/flex-layout